pages/TheLearnerView.html

```diff
@@ -51,7 +51,7 @@ <h3>
     dashboards of the exploration editors.
   </p>
 
-  <img src="static/images/explorationFeedback.png" width="500">
+  <img src="static/images/explorationFeedback.png" width="500" alt="An exploration card with a white background, along with a 'message' icon on the top right corner. On clicking this icon, a text area is displayed through which one can submit feedback.">
```

Comment:
I don't think people will understand / be able to visualize what an "exploration card" is.